docs/api/app.md

```diff
@@ -506,6 +506,41 @@ gets emitted.
 **Note:** Extra command line arguments might be added by Chromium,
 such as `--original-process-start-time`.
 
+### Event: 'notification-activation' _Windows_
```

Comment:
All the parameters to this event should be on a single `details` object, not each as an individual parameter

Reply:
I've changed event prototype and name of parameters https://github.com/electron/electron/commit/de1f5b59c626d2be2213c25d33ce105ddcdfa5e6